kuma/static/js/contribution-handler.js

```diff
@@ -399,10 +408,36 @@
         }
     }
 
+    /**
+     * Gets and executes stripe's checkout.js script to be used when submitting
+     * also handles errors when getting the resource
+     */
+    function getStripeCheckoutScript() {
+        $.getScript('https://checkout.stripe.com/checkout.js')
```

Comment:
I think if ``stripeHandler`` is initialized (not ``null``), then the script should not be reloaded. You can see the multiple loads by watching the network tab while collapsing and expanding the banner multiple times.

Reply:
Also, you should add the following config:

```
$.ajaxSetup({
  cache: true
});
```

If that is not set, then the script will be loaded over the network each time it is requested, instead of loading it from browser cache.

Reply:
Good suggestions, thank you. Please also see my comments around error handling improvements for the script loading as in the other comment as well.

Are we too consider this feedback launch blocking or is it something we can follow up with quickly next week?